application_sandbox/async_compute/main.cpp

```diff
@@ -974,26 +1036,6 @@ class AsyncSample : public sample_application::Sample<AsyncFrameData> {
     vulkan::MemoryClear(&clear);
     clear.color.float32[3] = 1.0f;
 
-    if (swapped_buffer) {
```

Comment:
I think `swapped_buffer` can be removed above, around line 944.

Reply:
Removed